todo_app/templates/index.html

```diff
@@ -2,16 +2,118 @@
 {% block title %}To-Do App{% endblock %}
 
 {% block content %}
-  <div class="jumbotron">
-    <h1 class="display-4">To-Do App</h1>
-    <p class="lead">Just another to-do app.</p>
+  <div class="jumbotron jumbotron-fluid" style="border-radius: 5px;">
+    <h1 class="display-4 ml-4">To-Do App</h1>
+    <hr class="my-6 ml-4 mr-4">
+    <p class="lead ml-4">Just another to-do app.</p>
   </div>
-
-  <div class="row">
-    <div class="col-md-12">
-      <h2>Items</h2>
-      <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+  <div>
+    <form action="/add_new" method="post">
+        <div class="input-group mb-3">
+          <input type="text" id="new_task" name="new_task" class="form-control" required placeholder="New Task" >
+          <div class="input-group-append">
+            <button class="btn btn-primary" type="submit" id="button" value="Submit">Add</button>
+          </div>
+        </div>
+    </form>
+  </div>
+  <div class="row-12 mb-4">
+    <div class="col-12 mb-3">
+      <ul class="list-group list-group-horizontal col-4 mb-3">
+        <div class="dropdown col-4">
+          <button class="btn btn-secondary dropdown-toggle" type="button" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+            Tasks
+          </button>
+          <div class="dropdown-menu mb-3" aria-labelledby="dropdownMenuButton">
+            <a class="dropdown-item" href="/view/all">All</a>
+            <a class="dropdown-item" href="/view/todo">To Do</a>
+            <a class="dropdown-item" href="/view/started">In Progress</a>
+            <a class="dropdown-item" href="/view/complete">Done</a>
+          </div>
+        </div>
+        {% if all %}
+        <div class="dropdown col-4">
+          <button class="btn btn-secondary dropdown-toggle" type="button" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+            Sort By
+          </button>
+          <div class="dropdown-menu mb-3" aria-labelledby="dropdownMenuButton">
+            <a class="dropdown-item" href="/sortby/todo">To Do -> Done</a>
+            <a class="dropdown-item" href="/sortby/done">Done -> To Do</a>
+          </div>
+        </div>
+        {% endif %}  
+      </ul>
+      <ul class="list-group mb-0">
+        <ul class="list-group col-12 mb-2">
+        {% for task in tasks %}
+          {% if 'Not Started' in task.status %}
```

Comment:
By doing the "if" up here, you have a lot of duplicated HTML. Looks like the only bits that vary between the different items are
- The status to display
- The form

The status to display should just be `{{ task.status }}` and then you should put the ifs around the possible forms.